source/iostreams.tex

```diff
@@ -11541,12 +11539,6 @@
 Template parameters named \tcode{Allocator} shall meet the
 \oldconcept{Allocator} requirements (\tref{cpp17.allocator}).
 
-\rSec3[fs.req.namespace]{Namespaces and headers}
```

Comment:
This part is present in the lwg2818 resolution, so not needed in the editorial changes, right?

Reply:
I think this entire pull request is mostly moot with LWG 2818 applied. I'll wait until that one has landed before reviewing for residual cleanup.

Reply:
LWG 2818 has landed!